src/App.js

```diff
@@ -1,63 +1,49 @@
+/*jshint esversion: 6 */
 import React, { Component } from 'react';
-import logo from './logo.svg';
 import './App.css';
-var https = require('https');
+var https = require('http');
```

Comment:
This should be changed back once we have a proper `https` domain.
```suggestion
var https = require('https');
```